### PR TITLE
Adding repository location to the addons tutorial

### PR DESCRIPTION
**Pull request by Mythobeast**
Adding repository location to the addons tutorial

### CODE REVIEW

File: developers/index.md [hunk 1]
```diff
@@ -76,6 +76,13 @@ We have prepared some step-by-step guides for the following IDEs:
 
 Not sure what to choose?: openHAB maintainers use [Eclipse IDE](https://wiki.eclipse.org/Eclipse_Installer).
 
+## Make your own copy of the code base
+
+OpenHAB has [several repositories](https://github.com/openhab) on GitHub. You can find a list of them in the Developer Appendix of this 
+documentation, in the [Contribuiting](https://www.openhab.org/docs/developer/contributing.html) section.
```
Comment (stefan-hoehn) [inline developers/index.md:82]: type in Contribuiting and please don't use absolute links. Use the following instead.

``
[Contributing](contributing.html) section.
``

**Review by stefan-hoehn — CHANGES_REQUESTED** [review]

File: developers/index.md [hunk 2]
```diff
@@ -76,6 +76,13 @@ We have prepared some step-by-step guides for the following IDEs:
 
 Not sure what to choose?: openHAB maintainers use [Eclipse IDE](https://wiki.eclipse.org/Eclipse_Installer).
 
+## Make your own copy of the code base
+
+OpenHAB has [several repositories](https://github.com/openhab) on GitHub. You can find a list of them in the Developer Appendix of this
```
Comment (florian-h05) [inline developers/index.md:81]:
```suggestion
openHAB has [several repositories](https://github.com/openhab) on GitHub.
The most important ones are listed in the [Contribution section of the Developer Appendix](contributing.html#the-repositories). 
```

File: developers/index.md (hunk 1 above)
Comment (florian-h05) [inline developers/index.md:82]:
```suggestion
```

Remove this line, see comment above.

File: developers/index.md [hunk 3]
```diff
@@ -76,6 +76,13 @@ We have prepared some step-by-step guides for the following IDEs:
 
 Not sure what to choose?: openHAB maintainers use [Eclipse IDE](https://wiki.eclipse.org/Eclipse_Installer).
 
+## Make your own copy of the code base
+
+OpenHAB has [several repositories](https://github.com/openhab) on GitHub. You can find a list of them in the Developer Appendix of this 
+documentation, in the [Contribuiting](https://www.openhab.org/docs/developer/contributing.html) section.
+
+For this tutorial, you'll want to clone the [OpenHAB-addons](https://github.com/openhab/openhab-addons) repository.
```
Comment (florian-h05) [inline developers/index.md:84]:
```suggestion
For this tutorial, you'll need to fork the [openhab-addons](https://github.com/openhab/openhab-addons) repository (press the fork button in GitHub) and then clone your fork.
```

**Comment by florian-h05** [comment]
Superseded by #2415.

@Mythobeast Thanks for your contribution, I have picked up your changes in #2415.
@stefan-hoehn Please close this PR.

**Comment by stefan-hoehn** [comment]
closing PR

**Pull request closed without merging**